template/cordova/lib/run.js

```diff
@@ -92,21 +152,26 @@ module.exports.help = function () {
     console.log('    run --target=7988B8C3-3ADE-488d-BA3E-D052AC9DC710');
     console.log('    run --device --release');
     console.log('    run --emulator --debug');
+    console.log('    run --device --appx=phone-8.1');
     console.log('');
+
     process.exit(0);
 };
 
 
 function getWindowsTargetVersion() {
     var config = new ConfigParser(path.join(ROOT, 'config.xml'));
-    var windowsTargetVersion = config.getPreference('windows-target-version');
+    var windowsTargetVersion = config.getWindowsTargetVersion();
     switch(windowsTargetVersion) {
-    case '8':
-    case '8.0':
-        return '8.0';
-    case '8.1':
-        return '8.1';
-    default:
-        throw new Error('Unsupported windows-target-version value: ' + windowsTargetVersion);
+        case '8':
+        case '8.0':
+            return '8.0';
+        case '8.1':
+            return '8.1';
+        case '10.0':
```

Comment:
Probably a fair comment.  I expect us to see more minor revisions over time (similar to how 8 and 8.1 were, although with the same project instead of different ones) and we designed it with that in mind.  If you want to target 10 base, you can use "UAP" here instead (aligned with the --appx=uap CLI option).